barretenberg/cpp/src/barretenberg/honk/library/grand_product_library.hpp

```diff
@@ -184,9 +184,10 @@ void compute_grand_product(typename Flavor::ProverPolynomials& full_polynomials,
 
     // For Ultra/Mega, the first row is an inactive zero row thus the grand prod takes value 1 at both i = 0 and i = 1
     if constexpr (IsUltraOrMegaHonk<Flavor>) {
```

Comment:
Is this conditional even meaningful anymore? maybe this code is used by the VMs or something but I didn't think so